e2e/testcases/cli_test.go

```diff
@@ -1034,3 +1034,19 @@ func checkFileExists(prefix string, targetFiles, allFiles []string) status.Multi
 	}
 	return err
 }
+
+func TestNomosStatus(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.NomosCLI)
+
+	// get status
+	cmd := nt.Command("nomos", "status")
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		nt.T.Log(string(out))
+		nt.T.Fatal(err)
+	}
+
+	if !strings.Contains(string(out), "git@test-git-server.config-management-system-test:/git-server/repos/config-management-system/root-sync/acme@main") {
```

Comment:
This breaks the CI jobs running against Bitbucket and Gitlab as the URLs are not the test-git-server. 
Link: https://oss.gprow.dev/view/gcs/oss-prow-build-kpt-config-sync/logs/multi-repo-1-standard-regular-bitbucket/1630776205252759552.

@victorpras Please file a RP to fix it.